content/en/security/code_security/static_analysis/ai_enhanced_sast.md

```diff
@@ -0,0 +1,196 @@
+| **Repository**         | Define which repo and paths to scan.         | - Set the GitHub repo URL<br>- Use **Paths** to limit rule scanning to certain directories or files                                                                                                                                                                                                      |
+| **Rule**               | Choose which SAST rule to apply.          | - Select a rule from the dropdown<br>- View description, code example, and number of matches<br>- Click **Show More** to see remediation steps                                                                                                                                                           |
+| **Session Management** | Controls how PRs are grouped and submitted.  | - **Create one PR per**:<br>      • `Repository`: One PR for all findings in the repo<br>      • `File`: One PR per file with findings<br>      • `Finding`: One PR per finding (most granular)<br>- **Allow [n] open PRs at a time**: Prevents too many PRs at once<br>- **Limit [n] findings per PR**: Prevents creating too-large PRs |
+| **Prompt Tuning**      | Customizes how the AI proposes remediations. | - **Custom Instructions**: Guide the AI on how to tweak fixes (for example, `Update CHANGELOG.md with a summary of changes`, `Start all PR titles with [autofix]`)                                                                                                    |
+### Campain in progress
+
+When you click **Create Campaign**, Bits Dev Agent does the following:
+
+1. Bits Dev Agent loads SAST findings for the selected repo(s), path(s), and rule.
+2. Uses the [Bits Dev Agent][10] to generate patches for each group of findings.
+3. Creates PRs according to your session rules.
+4. If Bits Dev notifications are turned on, PRs will automatically be sent via Slack to relevant stakeholders.
+5. Lets you review, edit, and merge fixes using GitHub.
+
+The campain page shows whether the AI is actively remediating real findings, and how many have been remediated or pending.
+
+You can click a session to view the code changes in more detail and chat with the [Bits Dev Agent][10] to ask for changes.
+
+### Session details
+
+A remediation session shows the full lifecycle of an AI-generated fix. It includes the original security finding, a proposed code change, an explanation of how and why the AI made the fix, and if enabled, CI results from applying the patch. 
+
+Session details make each remediation transparent, reviewable, and auditable, helping you safely adopt AI in your secure development workflow.
+
+Session details include the following:
```

Comment:
Visually it's a little weird that most of the items below are bold, but a few aren't (Header, Title, Bits chat field). If that's because of how they're labeled (or not) in the UI, IMO it'd be fine to just make them all bold anyway for consistency in this doc.